lib/src/main/kotlin/com/swmansion/starknet/data/types/Responses.kt

```diff
@@ -135,6 +135,8 @@ data class BlockWithTransactionsResponse(
     @SerialName("status")
     val status: BlockStatus,
 
+    // Block header
+
     @SerialName("parent_hash")
     override val parentHash: Felt,
```

Comment:
I just added this comment because in the RPC schema it is annotated that way. I think having comments about the schema structure instead of guessing where the flattened structure came from would simplify the process if there's ever a need to update any of it.
```json      
      "BLOCK_WITH_TX_HASHES": {
                "title": "Block with transaction hashes",
                "description": "The block object",
                "allOf": [
                    {
                        "title": "Block status",
                        "type": "object",
                        "properties": {
                            "status": {
                                "title": "Status",
                                "$ref": "#/components/schemas/BLOCK_STATUS"
                            }
                        },
                        "required": [
                            "status"
                        ]
                    },
                    {
                        "title": "Block header",
                        "$ref": "#/components/schemas/BLOCK_HEADER"
                    },
                    {
                        "title": "Block body with transaction hashes",
                        "$ref": "#/components/schemas/BLOCK_BODY_WITH_TX_HASHES"
                    }
                ]
            },
```

Reply:
I meant, since spec has a field `block_header` and not `parent_hash`, shouldn't we rename `parent_hash` to `block_header` instead having a comment?

Reply:
`block_header` is not a field in a schema, `BLOCK_WITH_TX_HASHES ` just expands fields from `block_header`.

Reply:
ah yes, you're right :+1: